src/ext/serial.cpp

```diff
@@ -25,18 +53,29 @@ class ThingSetStream: public ExtInterface
 
         virtual bool readable()
         {
+            #ifdef __MBED__
             return stream->readable();
+            #elif defined(__ZEPHYR__)
+            return true;
```

Comment:
There must be something to check if there are more characters in the buffer. Or if just returning true makes sense for some reason, please explain.